apps/dts-legacy/lgy-iac-web/dev.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: lgy-iac-web
+spec:
+  releaseName: lgy-iac-web
+  values:
+    java:
+      ingressHost: lgy-iac-web.dev.platform.hmcts.net
+      replicas: 1
```

Comment:
dev is not normally deployed via flux only via a pull request on Jenkins

Reply:
Thanks Tim - changing now

Reply:
Pushed changes